decidim-core/config/locales/en.yml

```diff
@@ -130,7 +130,14 @@ en:
         success: Your account was successfully deleted.
       show:
         available_locales_helper: Choose the language you want to use to browse and receive notifications on the platform
+        cancel_email_change_error: Could not cancel the email change.
+        cancel_email_change_successfully: Email change cancelled successfully.
         change_password: Change password
+        confirmation_email_resent_error: Could not resend confirmation email.
+        confirmation_email_resent_successfully: Confirmation email resent successfully to %{unconfirmed_email}
+        email_change_body: We've sent an email to %{unconfirmed_email} to verify your new email address.
+        email_change_send_again_or_cancel: Need us to send the verification email again? <a class="email-resend" href="%{resend_path}">Send again</a> or <a class="email-cancel-change" href="%{cancel_path}">Cancel</a>.
```

Comment:
I would prefer two changes here:
1. Move the links outside of the translation, just add a link "holder" interpolation to the translation
2. Change the links to `<button class="link">` elements, I believe they are better suited in this situation (for a11y reasons)

Reply:
1. Moved to link texts to their own translations, not totally sure if this what you meant..
2. This is why I made that js solution, because data-remote buttons would create nested forms, which I don't think is what we want.

Reply:
1. Yes this is what I meant but I would still keep all the texts in the same translation, as in:
```
Need us to send the verification email again? %{resend_link} or %{cancel_link}.
```
With some languages we may want to change the order of the strings a bit, which is why this kind of interpolation is generally better than hard coding the position for the "or" string.

2. Yes you can use but the problem you've probably faced is that when you add a `<button>` without "type" inside a form, it will default to a "submit" type button. But if you add `<button type="button">`, it won't submit the form by default.

Reply:
1. Done
2. This would create nested forms, added role="button" to links instead.

Reply:
1. Done
2. This would create nested forms, added role="button" to links instead.